lib/texutil.js

```diff
@@ -42,7 +42,29 @@ module.exports.latex_function_names = arr2set([
     "\\sin", "\\sinh", "\\sup", "\\tan", "\\tanh"
 ]);
 
-module.exports.paren = arr2set(["\\sinh"]); //incomplete, will be added later once CSV parser is finished
+module.exports.paren = arr2set(["\\sinh"]);
+try{ //try catch to make sure program still runs without csv file
+    var csv = require('csv-parse/lib/sync');
+    var fs = require('fs');
+    var path = require('path');
+    var filePathread = path.join(__dirname, 'optionalFunctions.csv');
```

Comment:
maybe we should make that filename configurable?
Will the commands be loaded during compile or during execution time?